EIPS/eip-3607.md

```diff
@@ -3,7 +3,7 @@ eip: 3607
 title: Reject transactions from senders with deployed code
```

Comment:
```suggestion
 title: Reject transactions from senders with deployed code
 description: Do not allow transactions for which `tx.sender` has any code deployed.
```
The `Simple Summary` section has been removed from the template and replaced with a `description` field in the frontmatter.  Along with this suggestion, please remove the Simple Summary section entirely.